sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -3,244 +3,121 @@
-      <EFDB_Name>annularZernikePoly</EFDB_Name>
-      <Description>z4-z22 terms of annular Zernike polynomials.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>micron</Units>
-      <Count>19</Count>
-    </item>
     <item>
-      <EFDB_Name>cameraRotation</EFDB_Name>
-      <Description>Camera rotation angle.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>userGain</EFDB_Name>
-      <Description>Gain value between 0 and 1. Set to -1 to ignore user gain.</Description>
-      <IDL_Type>double</IDL_Type>
+      <EFDB_Name>config</EFDB_Name>
+      <Description>Yaml configuration string with additional parameters to be passed to WEP process. If options are invalid command will be rejected. If not given, use the standard configuration.</Description>
+      <IDL_Type>string</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_processCmosWavefrontError</EFDB_Topic>
-    <Description>Process the wavefront data collection by CMOS camera.</Description>
-    <item>
-      <EFDB_Name>intraImgPath</EFDB_Name>
-      <Description>Absolute path to intra-focal wavefront image FITS file.</Description>
-      <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
+    <EFDB_Topic>MTAOS_command_runOFC</EFDB_Topic>
+    <Description>Apply user-provided wavefront errors.</Description>
     <item>
-      <EFDB_Name>extraImgPath</EFDB_Name>
-      <Description>Absolute path to extra-focal wavefront image FITS file.</Description>
-      <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>sglBrightStarPixelX</EFDB_Name>
-      <Description>Single bright star pixel x position. Put -99999 for the empty star.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>unitless</Units>
-      <Count>10</Count>
-    </item>
-    <item>
-      <EFDB_Name>sglBrightStarPixelY</EFDB_Name>
-      <Description>Single bright star pixel y position. Put -99999 for the empty star.</Description>
-      <IDL_Type>double</IDL_Type>
-      <Units>unitless</Units>
-      <Count>10</Count>
-    </item>
-    <item>
-      <EFDB_Name>filter</EFDB_Name>
-      <Description>Active filter type. Use 1-6 for u, g, r, i, z, and y. Use 7 for reference.</Description>
-      <IDL_Type>int</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-    <item>
-      <EFDB_Name>cameraRotation</EFDB_Name>
-      <Description>Camera rotation angle.</Description>
+      <EFDB_Name>wf</EFDB_Name>
+      <Description>Array with wavefront errors. Indexes are considered to be ANSI (https://en.wikipedia.org/wiki/Zernike_polynomials#OSA/ANSI_standard_indices).</Description>
```

Comment:
Do we need this argument when the `runWEP()` is applied already? The `runOFC()` can use the result from `runWEP()`. It is better to have some clear explanation here. When should we use this and when we do not need it.

Reply:
right. we definitely need more than 1x19 numbers...